test/datetypes.notz.test.js

```diff
@@ -0,0 +1,186 @@
+// Copyright IBM Corp. 2013,2016. All Rights Reserved.
+// Node module: loopback-connector-mysql
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+require('./init.js');
+var assert = require('assert');
+
+var db, DateModel;
+var mysqlVersion;
+
+
+
+var date = new Date(2015,10,25,13,44,55); //2015-12-25 13:44:55
+date = new Date();//(2015,10,25,13,44,55); //2015-12-25 13:44:55
+date.setMilliseconds(0);
+
+var dateOnly = new Date(2015,10,25); //2015-12-25
+var dateString = '2015-12-25 13:44:55';
+var dateOnlyString = "2015-12-25";
+
+
+function getTimeZone() {
+  var offset = new Date().getTimezoneOffset(), o = Math.abs(offset);
+  return (offset < 0 ? "+" : "-") + ("00" + Math.floor(o / 60)).slice(-2) + ":" + ("00" + (o % 60)).slice(-2);
+}
+
+var localTZ = getTimeZone();
+
+describe('MySQL DATE, DATETTIME, TIMESTAMP types on server with local TZ', function() {
```

Comment:
Please move these two test cases all under one test file. For example in [./test/datatypes.test.js/](https://github.com/strongloop/loopback-connector-mysql/blob/master/test/datatypes.test.js).